doc/installation.md

```diff
@@ -27,10 +27,9 @@ You will need:
 - Python 3
 - [Libvips](https://libvips.github.io/libvips/install.html)
 - pyvips (install it via python pip: `pip install pyvips`) 
-- [compose.py](https://github.com/CleverRaven/Cataclysm-DDA/blob/master/tools/gfx_tools/compose.py) script from the main [Cataclysm-DDA](https://github.com/CleverRaven/Cataclysm-DDA) repository
```

Comment:
why remove this line?

Reply:
ah well I thought we adopted the `compose.py` script. apparently not !

Reply:
So it is up to date, and last update to the one in the dda repo was also immediatly duplicated here, so I guess it's fine but it feels fragile